core/server/master/src/main/java/alluxio/master/file/meta/MutableInode.java

```diff
@@ -489,7 +487,40 @@ public T setUfsFingerprint(String ufsFingerprint) {
    * @return the updated object
    */
   public T setXAttr(Map<String, byte[]> xAttr) {
-    mXAttr = xAttr;
+    return setXAttr(xAttr, File.XAttrUpdateStrategy.TRUNCATE);
```

Comment:
Defaulting to use the `TRUNCATE` strategy as this is the same as the original behaviour.